docs/resources/instance_pool.md

```diff
@@ -43,7 +43,7 @@ The following arguments are supported:
 * `max_capacity` - (Optional) (Integer) The maximum number of instances the pool can contain, including both idle instances and ones in use by clusters. Once the maximum capacity is reached, you cannot create new clusters from the pool and existing clusters cannot autoscale up until some instances are made idle in the pool via [cluster](cluster.md) termination or down-scaling. There is no default limit, but as a [best practice](https://docs.databricks.com/clusters/instance-pools/pool-best-practices.html#configure-pools-to-control-cost), this should be set based on anticipated usage.
 * `idle_instance_autotermination_minutes` - (Required) (Integer) The number of minutes that idle instances in excess of the min_idle_instances are maintained by the pool before being terminated. If not specified, excess idle instances are terminated automatically after a default timeout period. If specified, the time must be between 0 and 10000 minutes. If you specify 0, excess idle instances are removed as soon as possible.
 * `node_type_id` - (Required) (String) The node type for the instances in the pool. All clusters attached to the pool inherit this node type and the pool’s idle instances are allocated based on this type. You can retrieve a list of available node types by using the [List Node Types API](https://docs.databricks.com/dev-tools/api/latest/clusters.html#clusterclusterservicelistnodetypes) call.
-* `custom_tags` - (Optional) (Map) Additional tags for instance pool resources. Databricks tags all pool resources (e.g. AWS & Azure instances and Disk volumes). *Databricks allows at most 43 custom tags.*
+* `custom_tags` - (Optional) (Map) Additional tags for instance pool resources. Databricks tags all pool resources (e.g. AWS & Azure instances and Disk volumes). The tags of the instance pool will propagate to the clusters using the pool (see the [official documentation](https://docs.databricks.com/administration-guide/account-settings/usage-detail-tags-aws.html#tag-propagation)). Attempting to set the same tags in both cluster and instance pool will raise an error. *Databricks allows at most 43 custom tags.*
```

Comment:
are you sure that the last sentence is correct regarding error? Documentation explicitly says:

> If there is a tag name conflict, Databricks default tags take precedence over custom tags and pool tags take precedence over cluster tags.

Reply:
That is at least what happens for me. Steps to reproduce
1. Create databricks_cluster resource with the following tags

```hcl
  instance_pool_id   = databricks_instance_pool.my_pool.id
  custom_tags = {
    application    = "dream-team"
    environment    = "dev"
} 
```

2. Create databricks_instance_pool my_pool
  custom_tags = {
    application    = "dream-team"
    environment    = "dev"
    }
    
I get this error on the cluster

│ Error: cannot update cluster:
│ Invalid tags - the following custom tag keys conflict with the pool's custom tag keys:

What happens if you try?

If the documentation says that this should work. Is this a bug in Terraform provider?

Reply:
Need to look - if it's behaviour change then we need to update docs.